ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -60,6 +60,7 @@ import {
 
 import Typography from '../../components/ui/typography/typography';
 import { MIN_GAS_LIMIT_DEC } from '../send/send.constants';
+import HelpOutLineIcon from '../../components/ui/icon/help-outline-icon.component';
```

Comment:
```suggestion
import HelpOutlineIcon from '../../components/ui/icon/help-outline-icon.component';
```

Reply:
fixed in 0fd7dc7f22cf45741ed4b85432f2dd1a2ea4ae83

Reply:
- Verified